index.html

```diff
@@ -1,182 +1,258 @@
-            dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+        <p class="text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
         </p>
-    </div>
-    
-    <ul class="lang">
+      </div>
+
+      <ul class="lang">
         <li class="css">css</li>
         <li class="liner">|</li>
         <li class="html">html</li>
         <li class="liner-2">|</li>
         <li class="boot">bootstrap</li>
         <li class="liner-3">|</li>
         <li class="ruby">Ruby</li>
-    </ul>
-    
-    <button class="see" type="button">See Project</button>
-    
-    <div class="project">
-        <img class="sample" src="./images/sample-image.png" alt="Project Sample Image">
+      </ul>
+
+      <button class="see" type="button">See Project</button>
+
+      <div class="project">
+        <img
+          class="sample"
+          src="./images/sample-image.png"
+          alt="Project Sample Image"
+        />
         <h3 class="title">Multi-Post Stories</h3>
-        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
-            dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+        <p class="text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
         </p>
-    </div>
-    
-    <ul class="lang">
+      </div>
+
+      <ul class="lang">
         <li class="css">css</li>
         <li class="liner">|</li>
         <li class="html">html</li>
         <li class="liner-2">|</li>
         <li class="boot">bootstrap</li>
         <li class="liner-3">|</li>
         <li class="ruby">Ruby</li>
-    </ul>
-    
-    <button class="see" type="button">See Project</button>
+      </ul>
 
- </section>
+      <button class="see" type="button">See Project</button>
+    </section>
 
-   
-</body>
-</html>
\ No newline at end of file
+    <section>
+      <div class="contact-form-container">
+        <h2 class="contacts-title">CONTACT US</h2>
+        <p class="contact-texts">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+      </div>
+
+      <div class="forms">
+        <form action="https://formspree.io/f/xdobrapq" method="post">
+          <ul>
+            <li>
+              <input
+                type="text"
+                placeholder="Text Input"
+                required
+                maxlength="30"
+              />
+            </li>
```

Comment:
- [ ] Kindly remove your `input` tags from the `ul` tags. It is best practise to wrap `input` tags in just a `form` tag 🙏

Reply:
Excuse me @Ugboaja-Uchechi The input tag is required to input the forms I have tried removing the input tags from the ul tags and it caused an error it didn't work. during the pair programming session that was the approach we used. Kindly assist me with a more detailed review of this review

Reply:
Hi @David-Kasilia , I don't mean you should remove the `input` tag completely from the code. What I mean is they shouldn't be wrapped inside of a `ul` tag. I just need you to remove the `ul` tag. Let me know if you have any more questions.

Reply:
Hello, @Ugboaja-Uchechi I tried to remove it actually alters my whole code on the forms flex container. If it is removed it changes the appearance of the code. Listings cannot be used without specifying the type of list. Kindly help me understand more about how to fix this issue.

Reply:
Remove the `ul` and `li` tags in your HTML file, after that go to your CSS file on line 282 make sure it is just `input` and not `form > ul > li > input`. Then you can adjust your code by yourself from there. Good luck 🤞 .

Reply:
@Ugboaja-Uchechi  I did that too. It did not work out as expected. The flex container is not working. I have tried to adjust the code too. What if the code was written by my coding partner shall I just change it?

Reply:
- I have taken into consideration your review and made the following changes:
- Removed the UI and li tags from the input tag.
- I adjusted my style.css code to make it more appealing.
- Here are the changes I made to the index.html regarding your review

![code](https://user-images.githubusercontent.com/83514256/177414443-df93d408-e1bc-4b4a-8759-984fcbfa5835.PNG)

Here is the preview after changing the code and adjusting the stylesheet

![preview](https://user-images.githubusercontent.com/83514256/177414452-8586308e-cdb6-49a5-8a2a-899bec10d621.PNG)
.